integration-cli/docker_api_containers_test.go

```diff
@@ -1730,3 +1736,48 @@ func (s *DockerSuite) TestContainersApiCreateMountsCreate(c *check.C) {
 		}
 	}
 }
+
+func (s *DockerSuite) TestContainersApiCreateMountsTmpfs(c *check.C) {
```

Comment:
@thaJeztah This is a new test. It should follow our coding standards.

Demonstrations of others not following the standards does not make it okay to not follow the standards.